desk/app/chat.hoon

```diff
@@ -1735,12 +1728,16 @@
   ::
   ++  cu-give-action
     |=  =action:club:c
+    =/  v5  (v5:action-club:v6:cv action)
```

Comment:
Same as I commented elsewhere, I think using a descriptive face with a version tag is better for readability.
```suggestion
    =/  action-5  (v5:action-club:v6:cv action)
```